drivers/modem/wncm14a2a.c

```diff
@@ -1283,9 +1240,8 @@ static int modem_pin_init(void)
 	 * When enabled, there will be no changes in the above 4 pins...
 	 */
 	LOG_DBG("SIG_TRANS_ENA_PIN -> ENABLED");
-	gpio_pin_set_raw(ictx.gpio_port_dev[SHLD_3V3_1V8_SIG_TRANS_ENA],
-			 pinconfig[SHLD_3V3_1V8_SIG_TRANS_ENA].pin,
-			 SHLD_3V3_1V8_SIG_TRANS_ENABLED);
+	gpio_pin_set_dt(&wncm14a2a_cfg.gpio[SHLD_3V3_1V8_SIG_TRANS_ENA],
+			SHLD_3V3_1V8_SIG_TRANS_ENABLED);
```

Comment:
```suggestion
	gpio_pin_set_dt(&wncm14a2a_cfg.gpio[SHLD_3V3_1V8_SIG_TRANS_ENA], 1);
```

suggestion: use plain `1`/`0`

Reply:
I like having the defines. It makes it easier for someone who has to change the IO logic. Maybe their design level 0 is enabled and now they only have to change the define.

Reply:
> I like having the defines. It makes it easier for someone who has to change the IO logic. Maybe their design level 0 is enabled and now they only have to change the define.

If level has to be controlled by the driver, `_raw` should be used. These defines can have different results depending on gpio flags in DT.

Reply:
so for my learning, `gpio_pin_set_dt(&wncm14a2a_cfg.gpio[SHLD_3V3_1V8_SIG_TRANS_ENA], 1);` doesn't just mean set the pin high, it depends on what is configured in the DT? So setting the pin could be setting it to 0 if that's how the DT is setup? If that's the case, then I am all for leaning on DT config to do this properly. That would mean any DT in the tree needs to be looked at to make sure it is setup correctly so this change does break operation.